bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -193,15 +193,9 @@ <h2 class="section-headline">{{ _('Our innovations') }}</h2>
 
         {% block icon_links_innovate %}
         <ul id="icon-links-innovate" class="icon-links">
-        {% if l10n_has_tag('home-update-20171116') %}
```

Comment:
I think we need to keep the l10n_has tag around the common voice stuff, we just don't display an else condition anymore.

Reply:
Ah, you are right. Added it back in. 👍